exercises/atbash-cipher/src/Atbash.hs

```diff
@@ -1,7 +1,7 @@
 module Atbash (decode, encode) where
 
 decode :: String -> String
-decode = error "You need to implement this function."
+decode phrase = error "You need to implement this function."
```

Comment:
I also prefer following the convention (`f` or `g` for functions, `xs` `ys` for generic lists, etc.) and I'm happy to change.

The reason I choose `phrase` here instead, was to match the test suite, believing that it would be easier for the student to understand the test cases.

I followed this same approach for other exercises, trying to match the test suite terms whenever possible, but I'm happy to change them as well.